e2e/cypress/tests/integration/integrations/incoming_webhook/attachment_does_not_collapse_spec.js

```diff
@@ -0,0 +1,73 @@
+            'Aliquam ornare sed tortor ut placerat. Fusce posuere a odio nec aliquet. Cras nec maximus metus. In elementum tincidunt orci, at sagittis nisl. Pellentesque scelerisque lorem ultricies ipsum finibus, in iaculis purus tincidunt. Aliquam tempus nunc at elementum vehicula. Integer tempus pretium magna, sed gravida nisl porta at. Donec et imperdiet augue, eget cursus dolor. Sed non magna dui. Phasellus vel massa pulvinar, cursus diam sit amet, vestibulum neque. Vivamus accumsan, mi vitae ultrices pretium, arcu eros sodales enim, et pellentesque quam eros in ligula. In eu justo a quam iaculis consequat. Aenean ornare a velit ac aliquet. Nullam lobortis posuere neque a pretium.' +
+            'Etiam dignissim sed ante commodo faucibus. Nunc vitae aliquet justo. Proin consequat leo vel libero porttitor, ac vulputate turpis bibendum. In vel libero sed odio euismod tincidunt id non dui. Quisque vitae est quis ante eleifend rutrum sed ut diam. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Integer rhoncus, leo nec iaculis sagittis, ligula mauris consequat felis, nec finibus odio sem tincidunt dolor. Pellentesque vel purus a sem rhoncus porta eget et erat. Sed interdum, justo ac dictum lacinia, nibh metus posuere arcu, vel euismod eros libero semper ligula. Proin elementum ligula quis ornare auctor. Integer vitae elementum augue, in congue lorem. Sed felis purus, consequat eu lacus in, fermentum accumsan diam. Duis lacus nunc, accumsan varius consequat nec, tincidunt et odio.';
+        const payload = {
+            channel: testChannel.name,
+            attachments: [{fallback: 'testing attachment does not collapse', pretext: 'testing attachment does not collapse', text: content}],
+        };
+        cy.postIncomingWebhook({url: incomingWebhook.url, data: payload, waitFor: 'attachment-pretext'});
+
+        // * Check "show more" button is visible and click
+        cy.getLastPostId().then((postId) => {
+            const postMessageId = `#${postId}_message`;
+            cy.get(postMessageId).within(() => {
+                cy.get('#showMoreButton').scrollIntoView().should('be.visible').and('have.text', 'Show more').click();
+            });
+        });
+
+        // # Type /collapse and press Enter
+        const collapseCommand = 'collapse';
+        cy.uiGetPostTextBox().type(`/${collapseCommand} {enter}`);
```

Comment:
nit: since no other use for `collapseCommand` variable, then it's fine to directly hardcode the text -- `/command {enter}`.

Reply:
Updated!